notify/wechat/wechat.go

```diff
@@ -134,16 +135,29 @@ func (n *Notifier) Notify(ctx context.Context, as ...*types.Alert) (bool, error)
 		n.accessTokenAt = time.Now()
 	}
 
+	// Define base message params
 	msg := &weChatMessage{
-		Text: weChatMessageContent{
-			Content: tmpl(n.conf.Message),
-		},
 		ToUser:  tmpl(n.conf.ToUser),
 		ToParty: tmpl(n.conf.ToParty),
 		Totag:   tmpl(n.conf.ToTag),
 		AgentID: tmpl(n.conf.AgentID),
-		Type:    "text",
```

Comment:
You can keep this line as well as the next one

```go
    Type = n.conf.MessageType,
    Safe: "0",
```

Reply:
According to the document, `markdown` mode doesn't support `safe` param.
So I move this param for `text` mode specifically.